cypress/e2e/keymaps.cy.js

```diff
@@ -314,4 +349,1450 @@ describe('Keymaps', () => {
       `);
     });
   });
+
+  describe('toggleComment', () => {
+    const blockStarter = `
+      <div>First line</div>
+      <div>Second line</div>
+      <div>Third line</div>`;
+
+    const lineStarter = `
+      <div
+        prop1="This is the first prop"
+        prop2="This is the second prop"
+        prop3="This is the third prop"
+      >
+        First line
+      </div>
+      <div>Second line</div>
+      <div>Third line</div>`;
+
+    const modifierKey = isMac() ? 'cmd' : 'ctrl';
+    const typeComment = () => typeCode(`{${modifierKey}+/}`);
```

Comment:
I initially interpreted this as "type a comment", but really it's doing "execute comment hotkey" or something of that nature.
```suggestion
    const executeCommentHotkey = () => typeCode(`{${modifierKey}+/}`);
```

Reply:
Yep good idea